src/shared/components/BeerCard/BeerCard.jsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import classNames from 'classnames';
 import Button from '../Button/Button';
-import './BeerCard.css';
+import './styles/BeerCard.scss';
 
 export default function BeerCard({ id, image, title, tagline, description, classNames: classes }) {
   const cardClass = classNames('card', classes, {
     'card_more-info': Boolean(description),
```

Comment:
you don't have to explicitly use `Boolean` here, if the description is falsy (undefined, null or empty line), then `classnames` will ignore this classname
[You can check it in docs here](https://jedwatson.github.io/classnames/)